src/components/Subheader/Subheader.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import useStyles from '../Subheader/Subheader.styles.js';
+import CloseIcon from '@material-ui/icons/Close';
+import PropTypes from 'prop-types';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+export default function Subheader(props) {
+  const { variant, mainLabel, subLabel, image, testClick } = props;
+  const isMobile = useMediaQuery('(max-width:1023px)');
+  const classes = useStyles({ variant, isMobile });
+  const [showSubheader, setSubheader] = useState(true);
+  const {
+    subheaderContainer,
+    subheader,
+    subheaderItems,
+    subheaderIcon,
+    mainText,
+    subText,
+    closeIcon,
+  } = classes;
+  const handleClick = () => {
+    setSubheader(!showSubheader);
+  };
```

Comment:
make it a one liner since you are not returning a value...

`const handleClick = () => setSubheader(!showSubheader);`

Reply:
Changed handleClick function to be more concise in commit -  5ec6c7f38e20fcde3947556935d9b744ce91061e